src/contracts/pods/EigenPod.sol

```diff
@@ -222,61 +239,52 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
         emit ValidatorRestaked(validatorIndex);
 
+        uint64 validatorEffectiveRestakedBalanceGwei = _calculateEffectedRestakedBalanceGwei(validatorCurrentBalanceGwei);
```

Comment:
function name typo(?):
`_calculateEffectedRestakedBalanceGwei` => `_calculateEffectiveRestakedBalanceGwei` ?

Reply:
actually maybe just eliminate the word `effective`, since this is their real "restaked balance", there's nothing "effective" about it.

Reply:
The idea with the effective is that, like ethereum, its some sort of modification/underestimation of the balance.  But ours is a pure underestimation unlike ethereum's.  Maybe good to get rid of